src/widgets/relevant-sort/relevant-sort.tsx

```diff
@@ -59,7 +61,7 @@ const renderer: Renderer<
 
   render(
     <RelevantSort
-      cssClasses={cssClasses}
+      cssClasses={cssClasses as RelevantSortComponentCSSClasses}
```

Comment:
how come this weirdness isn't needed in the other components? is it because the component only accepts component classes and TS doesn't know in the renderer that the classes are converted before they're passed? Maybe we need to specify in the WidgetParams of renderer that the classes are componentClasses?

Reply:
See https://github.com/algolia/instantsearch.js/pull/4774#discussion_r647230512